server/src/main/java/com/objectcomputing/checkins/services/questions/Question.java

```diff
@@ -0,0 +1,77 @@
+package com.objectcomputing.checkins.services.questions;
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.model.DataType;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+import java.util.UUID;
+
+
+@Entity
+@Table(name ="questions")
+public class Question {
+
+    public Question(@NotBlank @NotNull String text) {
```

Comment:
Only @NotBlank is needed.
https://javaee.github.io/javaee-spec/javadocs/javax/validation/constraints/NotBlank.html

Reply:
done